charts/pact-broker/values.yaml

```diff
@@ -498,3 +498,6 @@ serviceAccount:
 
   # -- Auto-mount the service account token in the pod
   automountServiceAccountToken: true
+
+  # -- Name of image pull secrets that should be attached to the service account
+  imagePullSecrets:
```

Comment:
this will need to be an empty list `[]` so when it's iterated over in the serviceaccount file, it should be able to use them as currently it believes this is a string